<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDialog</class>
 <widget class="QDialog" name="CSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BB Notifier Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CallNotifierServer.qrc">
    <normaloff>:/Images/Gear.png</normaloff>:/Images/Gear.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>4</y>
     <width>410</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane{
    border-top: 1px solid gray;
    border-bottom: 1px solid gray;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #FFFFFF, stop: 0.4 #E8E8E8,
                                stop: 0.5 #E0E0E0, stop: 1.0 #E8E8E8);

    margin-top: -1px;
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tabSettings">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QPushButton" name="btnGen">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>281</width>
       <height>155</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <item row="8" column="1">
       <widget class="QCheckBox" name="chkStartup">
        <property name="text">
         <string>Run at startup</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="edtPort">
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtPass">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblPass">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="chkShowNotificationPopup">
        <property name="text">
         <string>Show notification information popup</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="chkShowCallPopups">
        <property name="text">
         <string>Show call information popup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabDiagnostics">
    <attribute name="title">
     <string>Diagnostics</string>
    </attribute>
    <widget class="QPlainTextEdit" name="edtDiagnostics">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>371</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(250, 250, 250)</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diagnostics output (updating every 5 seconds):</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAbout">
    <attribute name="title">
     <string>About &amp;&amp; Info</string>
    </attribute>
    <widget class="QLabel" name="lblAutodiscovery_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Version:
    Server v.1.1.0
    (compatible with BB Notifier &lt;= v.1.1.0)</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>80</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
font: italic 10pt;
}</string>
     </property>
     <property name="title">
      <string>Notes</string>
     </property>
     <widget class="QLabel" name="lblAutodiscovery_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>This application requires BB Notifier app to be installed on your phone. You may download it here:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLinkLabel" name="linkLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Donate to future project development...
We have a lot of great ideas and features to implement !!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	font: 75 italic;
	text-decoration: underline;
}
</string>
      </property>
      <property name="text">
       <string>BB Notifier (Android) at Google Play Store</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblAutodiscovery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Autodiscovery packets are only sent while this dialog is opened.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lblAutodiscovery_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>227</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: Autodiscovery packets are only sent while this dialog is opened.</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLinkLabel</class>
   <extends>QLabel</extends>
   <header>LinkLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CallNotifierServer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
